test/testdata/app-scenarios/docker-compose/complex/docker-compose.imported.yaml

```diff
@@ -164,7 +164,13 @@ items:
           kind: ImageStreamTag
           name: db:latest
       type: ImageChange
-  status: {}
```

Comment:
And that as well in https://github.com/openshift/origin/pull/12143/commits/bbe3f65144d20a772b15da269d3641fe0ca70cd7.